README.md

```diff
@@ -1,6 +1,6 @@
 # build-database-ci-action
 
-This action uses [Flyway](https://flywaydb.org/) to spin up the specified database, run your migration scripts against it, and run your [tSQLt](https://tsqlt.org/) tests.
+This action uses [Flyway](https://flywaydb.org/) to spin up the specified database, run your migration scripts against it, and run your [tSQLt](https://tsqlt.org/) tests. 
```

Comment:
Is the only change in this a space at the end?

Reply:
Yep, needed something to commit so I added a space 😆